### ISSUE TITLE
Issue with extraFiles not being valid

### ISSUE THREAD

**Issue by mastergberry** [electron-userland/electron-builder#1302]
* **Version**: 14.3.0 (Previously on 13.5.0)

* **Target**: nsis

Getting the following error suddenly in the latest build

```
Error: Config is invalid:
{
  "extraFiles[2]": "Should be string",
  "extraFiles[3]": "Should be string",
  "extraFiles[6]": "Should be string",
  "extraFiles": [
    "Should be object",
    "Should be null,string",
    "Invalid option object"
  ],
  "extraFiles[0]": "Should be object",
  "extraFiles[1]": "Should be object",
  "extraFiles[4]": "Should be object",
  "extraFiles[5]": "Should be object"
}
```

This is the section of the config in question:

```
"extraFiles": [
      "lib/*.jar",
      "lib/Proguard/**/*",
      {
        "from": "lib/",
        "to": ".",
        "filter": [
          "*.dll"
        ]
      },
      {
        "from": "lib/",
        "to": ".",
        "filter": [
          "*.exe"
        ]
      },
      "licenses/**",
      "BLClient/BLClient.json",
      {
        "from": "include/",
        "to": "."
      }
    ],
```

@develar something recently break this behavior?

**Comment by mastergberry** [comment]
Looks like this commit might have changed the behavior?

https://github.com/electron-userland/electron-builder/commit/3fa62597ed85090ad1306d7124bf874f6edca726

**Comment by develar** [comment]
Will be fixed ASAP.

**Comment by develar** [comment]
Thanks! Fixed in upcoming 14.4.0
